src/classes/frDonation.cls

```diff
@@ -194,7 +238,7 @@ public class frDonation extends frModel implements frSyncable {
     }
    
     
-    private void createOpportunityMapping(Map<String, Object> request) {
+    public void createOpportunityMapping(Map<String, Object> request) {
```

Comment:
is there a reason we need to change this method to public?

Reply:
For visibility purposes to test in an NPC org, without it would need to make more modifications to change this but can look into it if needed!